doc/cygnus-ngsi/installation_and_administration_guide/performance_tips.md

```diff
@@ -36,6 +37,22 @@ Nevertheless, as explained above, it is highly recommended to increase at least
 
 [Top](#top)
 
+###<a name="section1.2"></a>Retries
+Batches could not be persisted. This is someting may occur from time to time because the sink is temporarilly not available, or the communications are failing. In that case, Cygnus has implemented a retry mechanism.
```

Comment:
"Batches could not be persisted" is maybe too bold. I'd suggest to use something softer, like: "It may happen that batches could not be persisted".